docs/usage/configuration-options.md

```diff
@@ -1845,6 +1845,30 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### overwriteSourceUrl
```

Comment:
Would it make things too complex internally if we simply used `sourceUrl`?

Reply:
Thats what I did first, but it is hard to maintain as I recall it just gets overwritten somewhere which I had no control of/in some other flow. so i just decided to have a new flag instead.

its a bit fuzzy as it was quite long time ago, but if needed i can try it again if needed.